pkg/chunk/disk_cache.go

```diff
@@ -26,21 +26,29 @@ import (
 	"math"
 	"os"
 	"path/filepath"
+	"reflect"
 	"regexp"
 	"runtime"
 	"sort"
 	"strings"
 	"sync"
 	"sync/atomic"
+	"syscall"
 	"time"
 
+	"github.com/davies/groupcache/consistenthash"
+	"github.com/google/uuid"
 	"github.com/juicedata/juicefs/pkg/utils"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/twmb/murmur3"
 )
 
 var (
-	stagingDir = "rawstaging"
-	cacheDir   = "raw"
+	stagingDir   = "rawstaging"
+	cacheDir     = "raw"
+	maxIODur     = time.Second * 5
```

Comment:
60 seconds

Reply:
ok, read/write on hdd bad sector may hang for 30s.

may be we should retry for 3 ~ 5 times ?

Reply:
change io timeout to 60s